test/fixtures/es-modules/pkgimports/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "imports": {
```

Comment:
we should add a percent encoded `/` in here somewhere.

Reply:
@bmeck I've added an explicit target test in the latest commit - let me know if that covers this case for you.